guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -34,6 +34,11 @@ Clients will remain supported.
 For more details, see the https://community.theforeman.org/t/drop-debian-11-ruby-2-7-and-nodejs-14-support-in-foreman-3-14/40503[removal RFC].
 endif::[]
 
+=== {awx} Parameter Change
```

Comment:
Is it a host parameter or a global setting?

Reply:
Perhaps worth to call out AAP 2.5+ compatibility? Not sure if AWX and AAP maintain the same versioning, but something like this perhaps?
```suggestion
===  Parameter change for {awx} 2.5+ compatibility
```

Reply:
I'm confused but I'm sure that Ewoud and other writers will help you to finish it.

Reply:
> Is it a host parameter or a global setting?

It's parameters that affect hosts. That means it can be a global parameter, but also a hostgroup parameter or a host parameter. However, not a global setting.

Reply:
> Perhaps worth to call out AAP 2.5+ compatibility? Not sure if AWX and AAP maintain the same versioning, but something like this perhaps?

I couldn't find any references to AAP in our documentation, so I wasn't sure if we should explicitly mention it. However, specifying the supported AAP versions would make this change clearer. Should we include it?

Reply:
That's why I didn't explicitly mention it. If I understood you correctly, we should explicitly mention versioning for AAP to ensure clarity.

Reply:
I think it helps readers, but wasn't quite sure how to best phrase it.